ProjectCoimbra.UWP/DataAccessLibrary/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("DataAccessLibrary")]
```

Comment:
We should add the MIT License header and ideally set these to values that make sense